internal/telemetry/collector_test.go

```diff
@@ -3,22 +3,24 @@ package telemetry_test
 import (
 	"bytes"
 	"context"
+	"encoding/json"
 	"fmt"
 	"runtime"
+	"sort"
 	"strings"
 	"testing"
 	"time"
 
+	"github.com/google/go-cmp/cmp"
```

Comment:
super tiny nit: go import grouping seems to be:

1. std library
2. same package (nginx/kubernetes-ingress)
3. third party (go-cmp/cmp should stay there)
4. test packages